src/main/java/lotto/domain/Reward.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+import lotto.exception.ErrorMessage;
+import lotto.exception.InvalidMatchCountException;
+
+public enum Reward {
+	THREE_MATCH_REWARD(3, 5000),
+	FOUR_MATCH_REWARD(4, 50000),
+	FIVE_MATCH_REWARD(5, 1500000),
```

Comment:
```suggestion
	FIVE_MATCH_REWARD(5, 1_500_000),
```
언더스코어를 활용하면 가독성을 좀 더 확보할 수 있습니다!

Reply:
요건 처음 알았네요....! 👍